testserv.c

```diff
@@ -383,6 +430,41 @@ cttest_peek_not_found()
     ckresp(fd, "NOT_FOUND\r\n");
 }
 
+void
+cttest_peek_ok_unix()
+{
+    progname = __func__;
+    char name[90];
+    snprintf(name, sizeof(name), "/tmp/ctb-%d", getpid());
+    mustforksrv(name);
+    assert(exist(name));
```

Comment:
These 5 lines above should be simplified. Please look at the ctdir() function. This functions returns path to the temp directory, so there is no need to do stuff with getpid(). I think that name for the socket path should be generated in the forking function.

Reply:
I initially wanted to to use that temporary directory, but I didn't search enough to figure out the `ctdir()` function was what I needed.